arch/loongarch/compare256_lasx.c

```
@@ -22,22 +22,18 @@ static inline uint32_t compare256_lasx_static(const uint8_t *src0, const uint8_t
 
         src0 += 32, src1 += 32, len += 32;
 
         ymm_src0 = __lasx_xvld(src0, 0);
         ymm_src1 = __lasx_xvld(src1, 0);
         ymm_cmp = __lasx_xvseq_b(ymm_src0, ymm_src1);
-        mask = (unsigned)lasx_movemask_b(ymm_cmp);
-        if (mask != 0xFFFFFFFF) {
-            uint32_t match_byte = (uint32_t)__builtin_ctz(~mask);
-            return len + match_byte;
-        }
+        mask = (uint64_t)lasx_movemask_b(ymm_cmp);
```

Cppcheck (2.19.0)

[style] The function 'lasx_sad_bu' is never used.

(unusedFunction)